css/types/color.json

```
@@ -460,6 +560,56 @@
             }
           }
         },
+        "color_function": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/color_value/color()",
+            "spec_url": "https://drafts.csswg.org/css-color-4/#icc-colors",
+            "description": "<code>color()</code>",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": "10.1"
+              },
+              "safari_ios": {
+                "version_added": "10"
```

The corresponding iOS version is 10.3. Or was this actually shipped in different versions on macOS and iOS? If you have a test page for this I or another reviewer could verify the exact versions using BrowserStack.

---

I have linked to a test page above, and I used Browserstack to test in Safari 10.

---

Thanks, I see [the test](https://result.dabblet.com/gist/f491f94dba0af1dfccffa24c46e770e5/7e79347a06ba4256297cc593dd24845d79efd0dd). BrowserStack has only Safari 10.1 for macOS, but Sauce Labs has both, and with that I could verify the test is red in 10 but green in 10.1. One cause for doubt is that it's not running the same macOS version, but it's probably right.

Verifying the iOS version is harder. ([For Safari for iOS, use the iOS version number, not the Safari version number or WebKit version number.](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#safari-for-ios-versioning)) Neither BrowserStack nor Sauce have iOS 10.x other than 10.3.

https://webkit.org/blog/10042/wide-gamut-color-in-css-with-display-p3/ links to https://trac.webkit.org/changeset/207442/webkit which landed *after* the release of iOS 10, so it seems 10.3 should be used then:

```suggestion
                "version_added": "10.3"
```